### DISCUSSION TITLE
Please approve my co-pilot request :(

### DISCUSSION THREAD

**Discussion by waldhari1** [community/community#9466] [Copilot Conversations] [answered]
Hello, 

I'm very much excited about beta testing co-pilot, please approve my account? i can't wait to start :(

**Comment by AhhhSaturn** [comment]
Patience, all will come in good time.

**Comment by davecheney** [comment, marked as answer]
Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
